build/config/compiler/BUILD.gn

```diff
@@ -514,7 +519,9 @@ config("compiler") {
     # http://crbug.com/427584
     cflags_cc += [ "-std=gnu++11" ]
   } else if (!is_win) {
-    cflags_cc += [ "-std=c++11" ]
+    cxx_flags = [ "-std=c++11" ]
```

Comment:
I don't think "cxx" is used as an abbreviation anywhere else in this file and it seems a bit out of place. maybe just call this 'cc_std' and move the +=s out of this else block?